integreat_cms/api/tests/api_tests_utils.py

```diff
@@ -0,0 +1,36 @@
+"""
+This modules contains helper functions for automatically generating view tests.
+"""
+import json
+import re
+
+
+def generate_test_functions(class_variables, endpoint_data):
+    """
+    This function generates test functions for a list of views.
+
+    :param class_variables: The class variable dict in which the functions should be inserted
+    :type class_variables: dict
+
+    :param endpoint_data: The list of endpoints for which the test functions should be generated
+    :type endpoint_data: list
+    """
+    for data in endpoint_data:
+        endpoint, wp_endpoint, fixture = data
```

Comment:
```suggestion
    for endpoint, wp_endpoint, expected_result in endpoint_data:
```
You can expand the parameters directly in the for loop.